src/components/screens/reclaimBalance/modal/status/status.test.js

```diff
@@ -80,12 +78,11 @@ describe('Status', () => {
 
     const wrapper = shallow(<Status {...propsWithError} />);
     expect(wrapper.find('.status-container')).toExist();
-    expect(wrapper.find(FailAction)).toExist();
     expect(wrapper.find(TransactionResult).props()).toMatchObject({
       illustration: 'default',
       status: { code: 'BROADCAST_ERROR', message: JSON.stringify({ message: 'error:test' }) },
       title: 'Transaction failed',
-      className: 'content false',
+      className: 'content',
```

Comment:
Can you add the rest of the text of the screen to text? I suspect error message is not passed.

Reply:
I do not pass the error. The `TransactionResult` component reads it from the Redux store.